quinn-proto/src/connection/receiver_timestamps.rs

```diff
@@ -0,0 +1,126 @@
+use std::{collections::VecDeque, fmt, time::Instant};
+
+use tracing::warn;
+
+#[derive(Debug, Clone, PartialEq, Eq, Copy)]
+pub struct PacketTimestamp {
+    pub packet_number: u64,
+    pub timestamp: Instant,
+}
+
+impl Default for PacketTimestamp {
+    fn default() -> Self {
+        Self {
+            packet_number: 0,
+            timestamp: Instant::now(),
```

Comment:
It's weird for `default()` to return different values over time. Maybe encode the delta here instead? Or, do we even need a `Default` impl?